static/app/views/issueList/searchBar.spec.tsx

```diff
@@ -165,4 +166,68 @@ describe('IssueListSearchBar', function () {
       );
     });
   });
+
+  describe('Tags and Fields', function () {
+    it('displays the correct options for the is tag', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/tags/',
+        body: [],
+      });
+
+      defaultProps.organization.features = ['issue-stream-search-query-builder'];
```

Comment:
If you mutate this here, subsequent tests will have the same value which you don't want. I'd pass a new organization prop instead